docs/data-builder-intro.md

```diff
@@ -76,7 +76,7 @@ Data Builder:
 * **Simplifies the implementation of new features** across multiple
   different data backends.
 
-For more information on how Data Builder and Cohort Extractor compare, 
+For more information on how Data Builder and Cohort Extractor compare,
```

Comment:
nit: This is a slightly redundant merge commit; my personal preference is to pull `main` and rebase a new branch on that. (You avoid the additional commit entirely then.) But not really a big deal in this repository's history.